src/app/Http/Kernel.php

```diff
@@ -61,6 +61,7 @@ class Kernel extends HttpKernel
         'signed' => \Illuminate\Routing\Middleware\ValidateSignature::class,
         'throttle' => \Illuminate\Routing\Middleware\ThrottleRequests::class,
         'verified' => \Illuminate\Auth\Middleware\EnsureEmailIsVerified::class,
+        'poll.role' => \App\Http\Middleware\CheckPollRole::class,
```

Comment:
you should call it poll.roles as to say that we can pass multiple roles as a param to that middleware

Reply:
Fixed in 1d8f0ba5bbadd30cf1d032c59dab32331b8b1dbd